BankSDK/GiniBankSDKExample/GiniBankSDKExample/AppCoordinator.swift

```diff
@@ -20,8 +20,7 @@ final class AppCoordinator: Coordinator {
         return selectAPIViewController
     }
     lazy var selectAPIViewController: SelectAPIViewController = {
-        let selectAPIViewController = (UIStoryboard(name: "Main", bundle: nil)
-            .instantiateViewController(withIdentifier: "selectAPIViewController") as? SelectAPIViewController)!
+        let selectAPIViewController = SelectAPIViewController()
```

Comment:
[Suggestion] Since we're doing a bit of refactoring here, what do you think about renaming `SelectAPIViewController` ?

Reply:
@zladzeyka what do you think about GiniDemoViewController or just DemoViewController?  or ExampleAppHomeViewController?

Reply:
`DemoViewController, HomeViewController` sounds good!